zmon-controller-app/src/main/java/org/zalando/zmon/controller/FeedbackController.java

```diff
@@ -0,0 +1,33 @@
+package org.zalando.zmon.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.zalando.zmon.domain.Feedback;
+import org.zalando.zmon.service.FeedbackService;
+
+@Controller
+@RequestMapping(value = "/rest")
+public class FeedbackController {
+
+    @Autowired
+    private FeedbackService service;
+
+    @Autowired
+    private ObjectMapper mapper;
+
+    @RequestMapping(value = "/feedback", method = RequestMethod.GET)
```

Comment:
`GET /rest/feedback` implies you are going to receive a list of "feedbacks" which is not the case here... I'd change it to something like `GET /rest/feedback-metadata` (or maybe feedback-info...)